src/slack-send.js

```diff
@@ -26,6 +28,18 @@ module.exports = async function slackSend(core) {
 
     let payload = core.getInput('payload');
 
+    const payloadFilePath = core.getInput('payloadFilePath');
+
+    if (payloadFilePath && !payload) {
+      try {
+        payload = await fs.readFile(path.resolve(payloadFilePath), 'utf-8');
+      } catch (error) {
+        // passed in payload file path was invalid
+        console.error(error);
+        throw new Error('Need to provide valid payload file path');
```

Comment:
Nice way to tell an error! How about being more specific?
```suggestion
        throw new Error(`The payload-file-path may be incorrect. Failed to load the file: ${payloadFilePath}`);
```